WordPress/Classes/Utility/ZendeskUtils.swift

```diff
@@ -85,6 +85,34 @@ import CoreTelephony
         })
     }
 
+    static func registerDevice(_ identifier: String) {
+        if ZDKConfig.instance().userIdentity == nil {
+            ZendeskUtils.createZendeskIdentity(withUser: false)
```

Comment:
Is this ever likely to happen, that we'll have no identity? Is there a reason we're creating an anonymous identity here instead of using `createIdentity` in `ZendeskUtils` to create one based on the user's site / accounts?

Reply:
Hey @frosty . When registering a device with ZD, the user information is irrelevant. ZD just needs the device ID and an instance of `ZDKConfig.instance().userIdentity`. It did happen while I was testing it. 

However, I just learned today that WP PNs require a .com account. So this may become moot anyway. I'll be making changes in light if my recent education.